src/App.js

```diff
@@ -1,5 +1,6 @@
 import logo from './logo.svg';
 import './App.css';
+import HelloComponent from './components/HelloComponents';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Planthor-Team/Planthor_ClientFrontEndWebApp/pullRequest?prid=7095961)
```suggestion
import HelloComponent from "./components/HelloComponents";
```